src/types/index.ts

```diff
@@ -149,16 +149,19 @@ export interface Grants {
     id: string;
 }
 
-export interface LiveSpecsExtBaseQuery {
+export interface LiveSpecsExtBareMinimum {
     catalog_name: string;
+    id: string;
+    spec_type: Entity;
+}
+
+export interface LiveSpecsExtBaseQuery extends LiveSpecsExtBareMinimum {
```

Comment:
Happy to see this interface paired down!

When we get some _free_ cycles, I would like to audit our queries across the board (e.g., tighten where needed, evaluate naming, reducing needless duplicate queries, etc.).

Reply:
Yeah... we should also take another look at upgrading Supabase so we can generate our types and use typing through their new SWR hook helper.